File: web/src/lib/database/storage_drives_types.ts
namespace _StorageDrivesTypes {
	export enum Column {
		ID = 'id',
		Description = 'description'
	}

	export const TableName = 'storage_drives_types'

	export interface Interface {
		id?: string[]
		description?: string[]
		storage_drive_type?: any[]
	}
}

export default _StorageDrivesTypes

File: internal/pkg/database/models/storage_drives_types.go
package models

type StorageDrivesTypes struct {
	ID          []string `sql:"primary_key" json:"id,omitempty"`
	Description []string `json:"description,omitempty"`
}

type storageDrivesTypesTable struct {
	TableName string

	ID               string
	Description      string
	StorageDriveType string
}

func StorageDrivesTypesTable() storageDrivesTypesTable {
	return storageDrivesTypesTable{
		TableName: "storage_drives_types",

		ID:          "id",
		Description: "description",
	}
}
